ReactiveFeedbackTests/SystemTests.swift

```diff
@@ -7,10 +7,10 @@ class SystemTests: XCTestCase {
 
     func test_emits_initial() {
```

Comment:
Should we restore the tests for the async version?

Reply:
Yeah 👍